crates/networking/p2p/kademlia.rs

```diff
@@ -292,37 +294,41 @@ impl KademliaTable {
         channels: PeerChannels,
         capabilities: Vec<Capability>,
     ) {
+        info!(
+            "[PEERS] Initializing backend communication with peer {:?}",
+            node_id
+        );
```

Comment:
Same here, while not as spammy, the information it provides is too specific to be useful as an info tracing

Reply:
Also, this method has way too much tracing in it